### PR TITLE
Alias string consistency

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #19856 reported by @MartijnMaandag 

### Summary of Changes
On the Tab 'Publishing' we do have the label 'Created by Alias' in Contacts, Banners and Articles (although in Contacts it is 'Created By Alias' with capital 'B').

**This PR makes them all "Created by Alias"**

In Tags, Newsfeeds and frontend content editing this is uses a generic string called 'Author's Alias'. 

**This PR makes the generic string "Created by Alias" and updates the generic description**

### CODE REVIEW

File: administrator/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -355,8 +355,8 @@ JGLOBAL_FIELD_CATEGORIES_CHOOSE_CATEGORY_DESC="Categories that are within this c
 JGLOBAL_FIELD_CATEGORIES_CHOOSE_CATEGORY_LABEL="Select a Top Level Category"
 JGLOBAL_FIELD_CATEGORIES_DESC_DESC="If you enter some text in this field, it will replace the Top Level Category Description, if it has one."
 JGLOBAL_FIELD_CATEGORIES_DESC_LABEL="Alternative Description"
-JGLOBAL_FIELD_CREATED_BY_ALIAS_DESC="Uses another name than the author's for display."
-JGLOBAL_FIELD_CREATED_BY_ALIAS_LABEL="Author's Alias"
+JGLOBAL_FIELD_CREATED_BY_ALIAS_DESC="Enter an alias to be displayed instead of the name of the user who created the item."
+JGLOBAL_FIELD_CREATED_BY_ALIAS_LABEL="Created by Alias"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.ini:359]: It should be `Created By Alias`. See 2 strings below `Created By`.

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.ini:359]: no it should be as I have submitted

Comment (Quy) [reply administrator/language/en-GB/en-GB.ini:359]: I see the 2 words having the same weight and together.

Comment (MartijnMaandag) [reply administrator/language/en-GB/en-GB.ini:359]: @Quy means line 361 has 'Created By' and maybe that must be 'Created by'? 
I don't know how it is in English, maybe @brianteeman can explain?

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.ini:359]: I agree that when it is just the two words then Capitalisation of both is correct
But when it is three words then the "by" is conjunctive and shouldnt be capitalised. 

Unfortunately there are no great rules on this sort of thing.

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.ini:359]: If I was writing "Created by Brian Teeman" then the "by" is just joining the two parts. But when I write "Created By" as a title then both parts are important

**Comment by Twincarb** [comment]
I have tested this item :white_check_mark: successfully on 7b0ac0296b671342b56020e572c2c9437473370a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19857">issues.joomla.org/tracker/joomla-cms/19857</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 7b0ac0296b671342b56020e572c2c9437473370a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19857">issues.joomla.org/tracker/joomla-cms/19857</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19857">issues.joomla.org/tracker/joomla-cms/19857</a>.</sub>

**Comment by infograf768** [comment]
II suggest to get rid of the "COM_CONTACT_FIELD_CREATED_BY_ALIAS_LABEL" and use in the xml the JGLOBAL_FIELD_CREATED_BY_ALIAS_LABEL string instead.
In that xml we already use some JGLOBAL strings.

**Comment by infograf768** [comment]
I am taking of the milestone and RTC until someone explains to me why the simplification I suggested above makes no sense.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19857">issues.joomla.org/tracker/joomla-cms/19857</a>.</sub>

**Comment by brianteeman** [comment]
Its really simple. The rules for J3 language files are that we should not delete any string. So even if we are not using the component specific string here it will still exist. Therefore there is no point in changing from the component specific string to the generic string.

**Comment by infograf768** [comment]
It's really simple: even if the string shall not be deleted (just leave it in this case without modification although I am pretty sure it is not used by any third party) it makes sense to change in core the xml to the global one and mark the Contact one as deprecated.

**Comment by brianteeman** [comment]
we cant be certain. these are your rules not mine. I wont comment any further as you have an agenda that is nothing to do with this PR

**Comment by rdeutz** [comment]
I don't know what are the rules we have to apply here, but using a global TAG make sense.

I have understand (understanding the fact) that we are not removing TAGS from ini files, so I can see a situation coming that someone changes a component ini file and wondering why the frontend string doesn't change. So at least we need a comment within the ini file to mark a Tag as not longer in use.

I would appreciate a clear written down set of rules for handeling language files.

**Comment by brianteeman** [comment]
If we want to have a global tag for this component then we should have it for all of them.

**Comment by infograf768** [comment]
When there is a global constant available, — and we are absolutely sure its value can't be translated differently depending on the place where it is used by any language —, it is indeed a very good idea to deprecate the local instance with the same value. As usual, and as decided previously, the deprecation should be done in next major release, i.e. 4.0.

**Comment by rdeutz** [comment]
@brianteeman can you make this PR again for 5.1, we discussed it today and accept the solution you have proposed. Thanks in advance.

**Pull request closed without merging**